codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/route53/Route53TrimHostedZone.kt

```diff
@@ -38,8 +39,7 @@ class Route53TrimHostedZone : SwiftIntegration {
         val inputShape = MiddlewareShapeUtils.inputShape(ctx.model, operationShape)
         val hostedZoneMember = inputShape.members().find { it.hasTrait<TrimHostedZone>() }
         if (hostedZoneMember != null) {
-            StripHostedZoneURLPathMiddleware.renderMiddleware(ctx, operationShape)
-            operationMiddleware.prependMiddleware(operationShape, StripHostedZoneUrlPathMiddlewareRenderable(ctx.model, ctx.symbolProvider))
+            operationMiddleware.prependMiddleware(operationShape, StripHostedZoneURLPathMiddlewareRenderable(ctx.model, ctx.symbolProvider))
```

Comment:
The middleware is no longer rendered for each operation that needs it since a reusable one is used instead.